### PR TITLE
feature/1772 Add slider inputs to choose bbox reference point for section enter events

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#1772

### CODE REVIEW

File: OTAnalytics/plugin_ui/line_section.py [hunk 1]
```diff
@@ -236,10 +245,15 @@ def _create_section(self) -> None:
         if self._metadata == {}:
             raise ValueError("Metadata of line_section are not defined")
         name = self._metadata[ID]
+        relative_offset_coordinates_enter = self._metadata[RELATIVE_OFFSET_COORDINATES][
+            "section-enter"
```
Comment (martinbaerwolff) [inline OTAnalytics/plugin_ui/line_section.py:249]: @briemla I think we need a constant here: Where should we define it? Can we take it from the enum class in types.py or do we have to define the constants somewhere else?

File: OTAnalytics/plugin_ui/toplevel_sections.py [hunk 2]
```diff
@@ -16,7 +17,18 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.title(title)
-        self.input_values: dict = {ID: ""} if input_values is None else input_values
+        # TODO: Get default values elsewhere!
+        self.input_values: dict = (
+            {
+                ID: "",
+                RELATIVE_OFFSET_COORDINATES: {
+                    "section-enter": {"x": 0.5, "y": 0.5},
```
Comment (martinbaerwolff) [inline OTAnalytics/plugin_ui/toplevel_sections.py:25]: See above

File: OTAnalytics/plugin_ui/toplevel_sections.py [hunk 3]
```diff
@@ -50,6 +73,9 @@ def _set_close_on_return_key(self) -> None:
 
     def close(self, event: Any = None) -> None:
         self.input_values[ID] = self.entry_name.get()
+        self.input_values[RELATIVE_OFFSET_COORDINATES][
+            "section-enter"
```
Comment (martinbaerwolff) [inline OTAnalytics/plugin_ui/toplevel_sections.py:77]: See above

**Review by briemla — CHANGES_REQUESTED** [review]
Looks good so far. Only small changes requested.

File: OTAnalytics/plugin_ui/toplevel_sections.py [hunk 4]
```diff
@@ -16,7 +18,18 @@ def __init__(
...
         super().__init__(**kwargs)
         self.title(title)
-        self.input_values: dict = {ID: ""} if input_values is None else input_values
+        # TODO: Get default values elsewhere!
+        self.input_values: dict = (
+            {
+                ID: "",
+                RELATIVE_OFFSET_COORDINATES: {
+                    EventType.SECTION_ENTER.serialize(): {"x": 0.5, "y": 0.5},
+                    "section-leave": {"x": 0.5, "y": 0.5},
```
Comment (martinbaerwolff) [inline OTAnalytics/plugin_ui/toplevel_sections.py:27]: @briemla do we have a global default defined elsewhere, which we can import here?

File: OTAnalytics/plugin_ui/toplevel_sections.py [hunk 5]
```diff
@@ -16,7 +18,18 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.title(title)
-        self.input_values: dict = {ID: ""} if input_values is None else input_values
+        # TODO: Get default values elsewhere!
+        self.input_values: dict = (
+            {
+                ID: "",
+                RELATIVE_OFFSET_COORDINATES: {
+                    EventType.SECTION_ENTER.serialize(): {"x": 0.5, "y": 0.5},
```
[the remark this answers is not in the document]
Comment (martinbaerwolff) [reply OTAnalytics/plugin_ui/toplevel_sections.py:26]: Thats right, in between I thought of makeing both available right away. Removed this

**Review by briemla — APPROVED** [review]

File: OTAnalytics/plugin_ui/toplevel_sections.py (hunk 4 above)
Comment (briemla) [reply OTAnalytics/plugin_ui/toplevel_sections.py]: Defined in adapter_ui.default_values

**Pull request merged**